### PR TITLE
Move config module from rtd-build repo

### PR DESCRIPTION
**Pull request by stsewd**
Still WIP

### CODE REVIEW

**Comment by stsewd** [comment]
Dependencies problem with `pytest-describe==0.11.0` :disappointed:

File: requirements/testing.txt [hunk 1]
```diff
@@ -2,8 +2,11 @@
 
 django-dynamic-fixture==2.0.0
 
-pytest<4,>=3.3.2
+# 3.6.1 and >3.2.5 is incompatible
+# with pytest-describe 0.11.0
+pytest==3.2.5
```
Comment (stsewd) [inline requirements/testing.txt]: `pytest-describe` is incompatible with the current pytest (`3.6.1`) had to downgrade to `3.2.5` :anguished:. We can port those tests later

**Comment by stsewd** [comment]
Not sure if this PR is big enough, but is ready. I think moving `config.py` is going to change a lot of files (here we need to change the imports of the existing files in rtd).

File: requirements/testing.txt (hunk 1 above)
Comment (agjohnson) [reply requirements/testing.txt]: Let's make an issue to address this later in the project.

Comment (stsewd) [reply requirements/testing.txt]: I opened an issue here https://github.com/rtfd/readthedocs.org/issues/4270

**Pull request merged**